frontend/src/scenes/feature-flags/FeatureFlagReleaseConditions.tsx

```diff
@@ -244,6 +244,7 @@ export function FeatureFlagReleaseConditions({
                                           })
                                         : null
                                 }
+                                exactMatchFeatureFlagCohortOperators={true}
```

Comment:
passing it into the FeatureFlagReleaseConditions component, which is the only place I need to do it (this component is reused when setting up feature flag targeting in surveys).